alibi_detect/cd/fet.py

```diff
@@ -108,3 +115,21 @@ def feature_score(self, x_ref: np.ndarray, x: np.ndarray) -> Tuple[np.ndarray, n
             odds_ratio[f], p_val[f] = fisher_exact(table, alternative=self.alternative)
 
         return p_val, odds_ratio
+
+    def get_config(self) -> dict:
+        """
+        Get the detector's configuration dictionary.
+
+        Returns
+        -------
+        The detector's configuration dictionary.
+        """
+        cfg = super().get_config()
```

Comment:
As much of the config as possible is populated in the parent classes `get_config`. Only the kwargs which are specific to a given detector subclass as populated in the subclasses `get_config` e.g. `alternative` in this case.